autogalaxy/galaxy/galaxy.py

```diff
@@ -182,6 +182,18 @@ def light_profile_linear_list(self) -> List[LightProfileLinear]:
     def has_light_profile_linear(self) -> bool:
         return len(self.light_profile_linear_list) > 0
 
+    @property
+    def light_profile_operated_list(self) -> List[LightProfileOperated]:
+        return [
+            value
+            for value in self.__dict__.values()
+            if is_light_profile(value) and isinstance(value, LightProfileOperated)

```

Comment:
I assume you don't need to check both that it is a light profile and that it is an instance of LightProfileOperated?